docs/docs/config/bluetooth.md

```diff
@@ -11,7 +11,7 @@ See [Configuration Overview](index.md) for instructions on how to change these s
 
 | Option                                 | Type | Description                                                                                                                                                                                                                                                             | Default |
 | -------------------------------------- | ---- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN`     | bool | Enables a combination of settings that are planned to be default in future versions of ZMK to improve connection stability. This includes changes to timing on BLE pairing initation, restores use of the updated/new LLCP implementation, and disables 2M PHY support. | n       |
+| `CONFIG_ZMK_BLE_EXPERIMENTAL_CONN`     | bool | Enables a combination of settings that are planned to be default in future versions of ZMK to improve connection stability. This includes changes to timing on BLE pairing initiation, restores use of the updated/new LLCP implementation, and disables 2M PHY support. | n       |
```

Comment:
Unfortunately this requires re-aligning the table, i.e. extending the width of this column by one in all other rows as well: https://github.com/zmkfirmware/zmk/actions/runs/8925194729/job/24520874985?pr=2290#step:4:137